src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -0,0 +1,95 @@
+
+    const nonce = getNonce();
+    html = html.replace(/\%nonce%/gi, nonce);
+    html = html.replace('%webview.cspSource%', webview.cspSource);
+    // necessary files from netron to work
+    html = this.updateExternalUri(html, webview, '%view-grapher.css%', 'view-grapher.css');
+    html = this.updateExternalUri(html, webview, '%view-sidebar.css%', 'view-sidebar.css');
+    html = this.updateExternalUri(html, webview, '%view-sidebar.js%', 'view-sidebar.js');
+    html = this.updateExternalUri(html, webview, '%view-grapher.js%', 'view-grapher.js');
+    html = this.updateExternalUri(html, webview, '%dagre.js%', 'dagre.js');
+    html = this.updateExternalUri(html, webview, '%base.js%', 'base.js');
+    html = this.updateExternalUri(html, webview, '%text.js%', 'text.js');
+    html = this.updateExternalUri(html, webview, '%json.js%', 'json.js');
+    html = this.updateExternalUri(html, webview, '%xml.js%', 'xml.js');
+    html = this.updateExternalUri(html, webview, '%python.js%', 'python.js');
+    html = this.updateExternalUri(html, webview, '%protobuf.js%', 'protobuf.js');
+    html = this.updateExternalUri(html, webview, '%flatbuffers.js%', 'flatbuffers.js');
+    html = this.updateExternalUri(html, webview, '%flexbuffers.js%', 'flexbuffers.js');
+    html = this.updateExternalUri(html, webview, '%zip.js%', 'zip.js');
+    html = this.updateExternalUri(html, webview, '%gzip.js%', 'gzip.js');
+    html = this.updateExternalUri(html, webview, '%tar.js%', 'tar.js');
+    // for circle format
+    html = this.updateExternalUri(html, webview, '%circle.js%', 'circle.js');
+    html = this.updateExternalUri(html, webview, '%circle-schema.js%', 'circle-schema.js');
+    // modified for one-vscode
+    html = this.updateUri(html, webview, '%index.js%', 'index.js');
+    html = this.updateUri(html, webview, '%view.js%', 'view.js');
+
+    return html;
+  }
+
+  private getMediaPath(file: string) {
+    return vscode.Uri.joinPath(this._extensionUri, CircleGraphCtrl.folderMediaCircleGraph, file);
+  }
+
+  private updateExternalUri(
+      html: string, webview: vscode.Webview, search: string, replace: string) {
+    const replaceUri = this.getUriFromPath(webview, CircleGraphCtrl.folderExternal + replace);
+    return html.replace(search, `${replaceUri}`);
+  }
+
+  private updateUri(html: string, webview: vscode.Webview, search: string, replace: string) {
+    const replaceUri = this.getUriFromPath(webview, replace);
+    return html.replace(search, `${replaceUri}`);
+  }
+
+  private getUriFromPath(webview: vscode.Webview, file: string) {
+    const mediaPath = this.getMediaPath(file);
+    const uriView = webview.asWebviewUri(mediaPath);
+    return uriView;
+  }
+};
+
+function getNonce() {
+  let text = '';
+  const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  for (let i = 0; i < 32; i++) {
+    text += possible.charAt(Math.floor(Math.random() * possible.length));
+  }
+  return text;
+}
```

Comment:
This code will be replaced by #679, right?

Reply:
Yup :)

Reply:
@seanshpark Do you want to merge it first? :)

Reply:
this and #679 does not depend on each other. it's ok with the order.